.circleci/config.yml

```diff
@@ -86,16 +86,16 @@ workflows:
           matrix:
             parameters:
               jobname:
-                - "py39-test-image-mpl334-cov"
```

Comment:
Need to regenerate the comparison images.

Reply:
You can do that as part of this PR by updating the hash.

Reply:
Where are these hashes stored?

Reply:
It is mentioned somewhere, maybe https://docs.astropy.org/en/latest/development/testguide.html#image-tests-with-pytest-mpl

@neutrinoceros recently did it so he would know best.

Reply:
that would be `astropy/tests/figures/` !